src/lcmap_rest/api/data/surface_reflectance.clj

```diff
@@ -42,10 +42,12 @@
   [band point time system]
   (let [[x y]   (point->pair point)
         times   (iso8601->datetimes time)
+        spec    (tile-db/find-spec band system)
         results (tile-db/find-tiles band x y times system)
         encoded (map #(assoc % :data (base64-encode (% :data))) results)]
     (log/info band x y times (count results))
-    (response {:result encoded})))
+    (log/info "GET tiles" band x y times (count results))
```

Comment:
I'm guessing both of these should be ``debug``, right? Probably not something useful during regular development, but only when we want to get deeper insight?